var/spack/repos/builtin/packages/cudnn/package.py

```diff
@@ -13,6 +13,10 @@ class Cudnn(Package):
 
     homepage = "https://developer.nvidia.com/cudnn"
 
+    version('7.6.3-10.1-x86_64', '463a4258c007599b42f04f5008116e40',
```

Comment:
Hi @bvanessen the checksums are now required to be sha256 vs. md5. Could you update them?